migration/migrating-3-4/migrating-openshift-3-to-4.adoc

```diff
@@ -1,7 +1,8 @@
 [id="migrating-openshift-3-to-4"]
 = Migrating {product-title} 3.x to 4.2
 include::modules/common-attributes.adoc[]
-:context: migrating-openshift-3-to-4
+:context: ocp-3-to-4
+:ocp-3-to-4:
```

Comment:
What is this attribute for? I don't see it used in any ifdef/ifeval statements or anything.